WordPress/src/main/java/org/wordpress/android/ui/compose/components/menu/dropdown/SubMenu.kt

```diff
@@ -0,0 +1,58 @@
+package org.wordpress.android.ui.compose.components.menu.dropdown
+
+import androidx.compose.foundation.background
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.MenuDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import me.saket.cascade.CascadeColumnScope
+import org.wordpress.android.ui.compose.theme.AppColor
+
+@Composable
+fun CascadeColumnScope.SubMenu(
+    element: MenuElementData.SubMenu,
+    onMenuItemClick: (MenuElementData) -> Unit,
+) {
+    val enabledContentColor = itemContentColor()
+    val disabledContentColor = if (isLightTheme()) {
+        AppColor.Gray10
+    } else {
+        AppColor.Gray50
+    }
+    DropdownMenuItem(
+        modifier = Modifier
+            .background(itemBackgroundColor()),
+        colors = MenuDefaults.itemColors(
+            textColor = enabledContentColor,
+            leadingIconColor = enabledContentColor,
+            trailingIconColor = enabledContentColor,
+            disabledTextColor = disabledContentColor,
+            disabledLeadingIconColor = disabledContentColor,
+            disabledTrailingIconColor = disabledContentColor,
+        ),
+        text = {
+            Text(
+                text = element.text,
+                color = enabledContentColor,
```

Comment:
❓ Will the `textColor` coming from `DropdownMenuItem` be applied properly if you set a color directly in this `Text` composable?

Reply:
Good catch, thanks! Removed `color` from `Text`